guide.md

```diff
@@ -11908,7 +12026,7 @@ Offline versions (best format for the best readability) of this guide at:
 
 -   PDF: <https://anonymousplanet-ng.org/export/guide.pdf> <sup>[[Archive.org]][726]</sup> <sup>[[Tor Mirror]][727]</sup>
 
--   OpenDocument Text (ODT) version at: <https://anonymousplanet-ng.org/export/guide.odt> <sup>[[Archive.org]][732]</sup> <sup>[[Tor Mirror]][733]</sup>
+-   <del>OpenDocument Text (ODT) version at: <https://anonymousplanet-ng.org/export/guide.odt> <sup>[[Archive.org]][732]</sup> <sup>[[Tor Mirror]][733]</sup></del> (temporarily unavailable)

```

Comment:
Here:

https://anonymousplanet-ng.org/export/guide.odt
https://web.archive.org/web/20220609222023/https://anonymousplanet-ng.org/export/guide.odt

Reply:
Ah, it is outdated. I think focus has been on the Markdown and the PDF not the ODT, or any other format possible with pandoc.